templates/doubled_report.html

```diff
@@ -0,0 +1,166 @@
+{% extends 'base/b4_base_with_nav.html' %}
+
+{% load staticfiles %}
+
+{% block content %}
+
+<div class="container-fluid">
+    <div class="alert-container"></div>
+
+    <h4 class="text-align-center" style="margin-bottom: 25px;">Students assigned to many classes</h4>
```

Comment:
Are we using bootstrap 3 or 4? If 4, there are utility classes for margins and padding (https://getbootstrap.com/docs/4.1/utilities/spacing/)